webapp/graphql/Search.js

```diff
@@ -1,15 +1,17 @@
 import gql from 'graphql-tag'
-import { userFragment, postFragment } from './Fragments'
+import { userFragment, postFragment, tagsCategoriesAndPinnedFragment } from './Fragments'
```

Comment:
Why do we need the tagsCategoriesAndPinnedFragment here?
It leeds the cypress tests to fail and I cannot see any reason why we would need this information for the search results.